sql/wsrep_sst.cc

```diff
@@ -1230,6 +1230,8 @@ int wsrep_remove_sst_user(bool initialize_thread) {
                         nullptr,
                         "SET SESSION lock_wait_timeout = 1;",
                         nullptr,
+                        "REVOKE mysql.pxc.sst.role FROM 'mysql.pxc.sst.user'@'localhost';",
```

Comment:
Why is this needed?

Reply:
We discuss this.
The correct way to remove the user when having a role, is to revoke the role and then drop the user.
Just dropping the user will work, but the role cleanup is delegated to MySQL, and as we have identified with mtr tests on very fast machines this looks like to be asynchronous. 
So to avoid problems and to be on the safe side, we should operate following the right execution. Role first then drop user.